examples/udp_server.rb

```diff
@@ -0,0 +1,60 @@
+require 'bundler/setup'
+require 'concurrent/actor'
+
+
+class MyUDPServer < Concurrent::Actor::RestartingContext
+  @@thread = {}
```

Comment:
I couldn't figure out a way to properly exit the thread if the actor dies so I tried another road which was to keep the receiving thread linked to the socket alive and only restart the actor itself, the thread variable is used so that the thread is only spawned once.

As I said in my pr message I not that entirely happy with this solution but I tested it and at least it works, at least from what I remember when I tried :)